.github/workflows/ci.yml

```diff
@@ -909,16 +909,21 @@ jobs:
         working-directory: tokio
 
   wasm32-wasi:
-    name: wasm32-wasi
+    name: ${{ matrix.target }}
     needs: basics
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - wasm32-wasi
+          - wasm32-wasi-preview1-threads
```

Comment:
How do these target names interact with #6484?

Reply:
we can change the `wasm32-wasi-preview1-threads` to `wasm32-wasip1-threads` after the name changed